lua/nwlib/init.lua

```diff
@@ -0,0 +1,102 @@
+AddCSLuaFile("cl_init.lua")
+
+local plymeta = FindMetaTable("Player")
```

Comment:
I suggest using `local plymeta = assert(FindMetaTable("Player"), "FAILED TO FIND PLAYER TABLE")